backend/pkg/metrics/README.md

```diff
@@ -16,17 +16,17 @@ kubeclarity:
 
 The metrics that are exposed are:
 
-| Metric                                               | Explanation                                                        |
-|------------------------------------------------------|--------------------------------------------------------------------|
-| kubeclarity_application_vulnerability                | Count of vulnerabilities per application, environment and severity |
-| kubeclarity_number_of_applications                   | The total number of applications                                   |
-| kubeclarity_number_of_fixable_vulnerabilities        | The number of fixable vulnerabilities per severity                 |
-| kubeclarity_number_of_fixable_vulnerabilities_amount | The total number of fixable vulnerabilities                        |
-| kubeclarity_number_of_packages                       | The total number of packages                                       |
-| kubeclarity_number_of_resources                      | The total number of resources                                      |
-| kubeclarity_number_of_vulnerabilities                | The number of vulnerabilities per severity                         |
-| kubeclarity_number_of_vulnerabilities_amount         | The total number of vulnerabilities                                |
-| kubeclarity_vulnerability_trend                      | Vulnerability trend in a 60 minute time window                     |
+| Metric                                               | Explanation                                                                          |
+|------------------------------------------------------|--------------------------------------------------------------------------------------|
+| kubeclarity_application_vulnerability                | Count of vulnerabilities per application, environment and importance (i.e. severity) |
+| kubeclarity_number_of_applications                   | The total number of applications                                                     |
+| kubeclarity_number_of_fixable_vulnerabilities        | The number of fixable vulnerabilities per importance (i.e. severity)                 |
```

Comment:
The industry standard for vulnerabilities severity is severity so it would be nice to avoid using a different word to describe it, I personally don't know enough about prometheus to know the answer here, but can we have a label like "vul_severity"? If not "seriousness" might be a better descriptor than importance here I think.